### DISCUSSION TITLE
How to avoid the warning: Warning: ReactDOM.render is no longer supported in React 18. Use createRoot instead. Until you switch to the new API, your app will behave as if it's running React 17. Learn more: https://reactjs.org/link/switch-to-createroot

### DISCUSSION THREAD

**Discussion by piyushjoshi1016** [infinitered/ignite#2464] [Q&A]
I am new to Ignite and wondering that how can I resolve this warning message. I tried few things but it seems like ReactDOM is controlled by react native library and we cannot change it through Ignite boilerplate. Please suggest the solution for this problem.

index.js:1 Warning: ReactDOM.render is no longer supported in React 18. Use createRoot instead. Until you switch to the new API, your app will behave as if it's running React 17. Learn more: https://reactjs.org/link/switch-to-createroot.

Also getting this warning:
index.js:1 WebSocket connection to 'ws://localhost:9090/' failed: 

Any help would be appreciated.  

Thank you

**Comment by joshuayoes** [comment]
Expo manages Ignite's entry point into React via registerRootComponent in App.js. Other people have had issues with this in the past, check out this issue for some solutions.
https://github.com/expo/expo/issues/18485
